pepper-apis/src/main/antlr4/org/broadinstitute/ddp/pex/lang/Pex.g4

```diff
@@ -72,6 +72,7 @@ predicate
 // Queries to pull out various pieces of profile data
 profileDataQuery
   : 'birthDate' '(' ')'   # ProfileBirthDateQuery
+  | 'isGovernedParticipant' '(' ')' #ProfileIsGovernedParticipantQuery
```

Comment:
🤔 I think it will make more sense to have this in `StudyQuery` instead of `ProfileQuery`, so something like:
```
user.studies["X"].isGovernedParticipant()
```
What this says is, "is user Y a governed participant in study X?" Their governance status doesn't feel like a piece of profile data, but rather a piece of study data. In fact, in the future, it might be possible that a participant is a governed participant in one study but not governed in another study.

Reply:
You're right, the governance is something under the study. But in this case, it checks whether operator is doing something on behalf of governed user or as a self-enrolled user.

Reply:
I mean, it checks the context of operation, so study is redundant.

Reply:
Maybe attaching this to `profile` wasn't good idea either... What do you think?

Reply:
I think it makes more sense to put it in `StudyQuery`, as I outlined in the first comment.

Reply:
moved to studyQuery